modules/account-lib/src/coin/eth/transactionBuilder.ts

```diff
@@ -553,6 +562,15 @@ export class TransactionBuilder extends BaseTransactionBuilder {
     this._forwarderAddress = address;
   }
 
+  /**
+   * Enable v field to be setting in all transactions.
+   *
+   * @param {boolean} [enable=true] the contract address of the token to flush
+   */
+  enableFinalVField(enable = true): void {
+    this._enableFinalVField = enable;
```

Comment:
please explain why we need this in the comments and then in the sub-comments under celo

Reply:
done